misc/ls-extensions/modules/sa-modeling-service/src/main/java/io/ballerina/componentmodel/servicemodel/nodevisitors/StatementNodeVisitor.java

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's change the license headers in all the newly added files as follows. 
```suggestion
 *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

Reply:
Updated.